src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -10813,24 +10817,26 @@ private int Foo(int i)
         return 1;
     }
 }";
-            var expected = @"
+            var expected = $@"
 class Program
-{
+{{
     private void Bar()
-    {
-        Foo(Foo(););
-    }
+    {{
+        Foo(Foo(){commitChar});
```

Comment:
This is the correct behavior for `.` at this location, but it is not the correct behavior for `;`. I'm guessing the difference is caused by not having the Complete Statement command handler hooked up for this test.

Reply:
I think this is the unit test for SymbolCompletionProvider, I could add this to the CompletionCommandHanldler test